src/components/Header.tsx

```diff
@@ -73,7 +73,13 @@ const Header = () => {
         loading={!blockHeight}
       />
       <HeaderItem
-        value={gateways ? Object.keys(gateways).length : undefined}
+        value={
+          gateways
+            ? Object.entries(gateways).filter(
+                (entry) => entry[1].status === 'joined',
```

Comment:
```suggestion
                (_address, gateway) => gateway.status === 'joined',
```

Reply:
I tried something like this initially but I get a lint error on unused _address. I went ahead with the change but added a //eslint-disable